recipes/bob.db.avspoof/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "2.0.6" %}
+
+package:
+  name: bob.db.avspoof
+  version: {{ version }}
+
+source:
+  fn: bob.db.avspoof-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.db.avspoof/bob.db.avspoof-{{ version }}.zip
+  md5: dcf5285054a2abeb9ba5c24849ca0789
+
+build:
+  entry_points:
+    - debug_avspoof = bob.db.avspoof.debug_avspoof:main
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
+    - sqlalchemy
+    - bob.io.base
+    - bob.db.base
+    - antispoofing.utils
+    - bob.db.verification.utils
+
+  run:
+    - python
+    - six
+    - sqlalchemy
+    - bob.io.base
+    - bob.db.base
+    - antispoofing.utils
+    - bob.db.verification.utils
+
+test:
+  commands:
+    - bob_dbmanage.py avspoof download
```

Comment:
downloads the file list of the database which is in a sql database. The database is required for nosetests.